docs/extensibility/theme-support.md

```diff
@@ -151,3 +151,11 @@ body.gutenberg-editor-page .editor-block-list__block[data-align="full"] {
 You can use those editor widths to match those in your theme. You can use any CSS width unit, including `%` or `px`.
 
 Further reading: [Applying Styles with Stylesheets](https://wordpress.org/gutenberg/handbook/blocks/applying-styles-with-stylesheets/).
+
+## Styling blocks
```

Comment:
Maybe this should be "Default Block Styles" or something more obvious. If it were, you could omit the bolding I did in my tweaks below.

Reply:
Agreed. I updated to your suggestion. The title should have been more focused but was leftover from an earlier PR that documented more than just default styles.